homework/05LessonFive.js

```diff
@@ -16,27 +16,33 @@ var six = 6;
 
 // Problem 1:
 // What is the result of applying the increment (++) operator to var one?
-var onePlusPlus;
+one++;
+var onePlusPlus = one;  //2
 
 // Problem 2:
 // What is the result of applying the decrement (--) operator to var two?
-var twoMinusMinus;
+two--;
+var twoMinusMinus = two; //1
 
 // Problem 3:
 // What is the result of three += 3?
-var threePlusAssignmentThree;
+three+=3;
+var threePlusAssignmentThree = three; //6
 
 // Problem 4:
 // What is the result of four -= 3?
-var fourMinusAssignmentThree;
+four -=3;
+var fourMinusAssignmentThree = four; //1
 
 // Problem 5:
 // What is the result of five *= 2?
-var fiveMultiplicationAssignmentTwo;
+five *=2;
+var fiveMultiplicationAssignmentTwo = five; //10
 
 // Problem 6:
 // What is the result of six /= 3?
-var sixDivisionAssignmentThree;
+six /=3;
+var sixDivisionAssignmentThree = six; //2
```

Comment:
Assign a literal numerical value to each problem variable to prove you understand what `*=` and `/=` is actually doing.